GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -1,8 +1,28 @@
 import GliaCoreSDK
 
 extension Glia {
+    /// Deprecated, use ``Glia.configure(with:uiConfig:assetsBuilder:completion:)`` instead.
```

Comment:
I put in the Core SDK deprecation that you should use the completion that has a result. Perhaps it would be useful because they have the same parameter names so the integrator could be confused.

Reply:
Fixed e43442f087028b2bc17b92c31a1ad7fee353221c